scripts/update_volume_monitor_aggressiveness.py

```diff
@@ -78,7 +78,12 @@ def update_aggressiveness_retrain_and_enable(monitors):
         ]:
             continue
 
-        if monitor["config"]["symptom"]["aggressiveness"]["level"] != 7:
+        # use auto-discovered symptom if present
+        symptom_config = monitor["status"].get("runtimeConfig", {}).get("symptomConfig")
+        if not config:
```

Comment:
if not `symptom_config`

Reply:
Good catch. I also caught it during testing